src/navs/index.js

```diff
@@ -179,6 +175,14 @@ export const handbookSidebar = [
                 name: 'Feedback',
                 url: '/handbook/people/feedback',
             },
+            {
+                name: 'Clubs',
+                url: '/handbook/people/clubs',
+            },
+            {
+                name: 'Lore',
```

Comment:
I don't think I was super familiar with the term "lore" before James used it. (Ie: I don't believe it's in America's vocabulary.) I wonder if there's another term we could use here?

Reply:
It's a very common phrase. Definitely not a British-ism, but suggestions welcome.

Reply:
+1 to Andy. At most, it is a bit nerd-coded.